config/default/kustomization.yaml

```diff
@@ -10,11 +8,7 @@ namespace: capi-system
 # field above.
 namePrefix: capi-
 
-# Labels to add to all resources and selectors.
```

Comment:
I would retain this, or if we want to remove it, do it in a separate PR. WDYT?

Reply:
Removing it in a separate PR is fine with me.